hil-test/Cargo.toml

```diff
@@ -221,6 +225,7 @@ defmt              = "0.3.8"
 defmt-rtt          = { version = "0.4.1", optional = true }
 embassy-executor   = "0.7.0"
 embassy-futures    = "0.1.1"
+embedded-storage   = { version = "0.3.1", optional = true }
```

Comment:
Just pull this in unconditionally, it's light, and the feature to enable it just clutters the test cases.

Reply:
👍 I also guess we should make esp-storage have inherent functions and make it useable without the traits - but something for another PR